src/test/java/junit/edu/umich/its/cpmtest/test.properties

```diff
@@ -0,0 +1,3 @@
+ctools.server.url=http://google.com
+username=test_user
+password=eeeee
```

Comment:
Just so nobody thinks checking in a password is ok, how about making it a password that nobody will ever think is real?  E.g. MY_BEAUTIFUL_BUT_USELESS_NONPASSWORD?